db/schema.rb

```diff
@@ -179,17 +179,32 @@
   add_index "user_preferences", ["user_id"], name: "index_user_preferences_on_user_id", using: :btree
 
   create_table "users", force: :cascade do |t|
-    t.string   "login",      limit: 40
-    t.string   "name",       limit: 100, default: ""
-    t.string   "email",      limit: 100
+    t.string   "login",                  limit: 40
+    t.string   "name",                   limit: 100, default: ""
+    t.string   "email",                  limit: 255, default: "",          null: false
     t.datetime "created_at"
     t.datetime "updated_at"
-    t.string   "password",   limit: 40,                        null: false
-    t.string   "city",       limit: 255
-    t.string   "token",      limit: 16
-    t.string   "role",       limit: 255, default: "contester", null: false
-  end
-
+    t.string   "password",               limit: 40,                        null: false
+    t.string   "city",                   limit: 255
+    t.string   "token",                  limit: 16
+    t.string   "role",                   limit: 255, default: "contester", null: false
+    t.string   "encrypted_password",     limit: 255, default: "",          null: false
+    t.string   "reset_password_token",   limit: 255
+    t.datetime "reset_password_sent_at"
+    t.datetime "remember_created_at"
+    t.integer  "sign_in_count",          limit: 4,   default: 0,           null: false
+    t.datetime "current_sign_in_at"
+    t.datetime "last_sign_in_at"
+    t.string   "current_sign_in_ip",     limit: 255
+    t.string   "last_sign_in_ip",        limit: 255
+    t.string   "provider",               limit: 255
+    t.string   "uid",                    limit: 255
```

Comment:
20151129133445_add_omniauth_to_users.rb add provider and uid, the rest firels are from initial generation of user model.